GooglePlayInstant/LoadingScreen/LoadingBar.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GooglePlayInstant.LoadingScreen
+{
+    /// <summary>
+    /// Class that encapsulates the creation and update of a Loading Bar component in the editor and during the game's
+    /// runtime.
+    /// </summary>
+    public static class LoadingBar
```

Comment:
It's strange that this is a static class, since its something that lives in the scene and gets updated. Maybe try splitting up the LoadingBar creation code into a LoadingBarFactory (working title) and a LoadingBar class that inherits from MonoBehaviour.